### PR TITLE
Add basic file structure

### PR DESCRIPTION
**Pull request by eurunuela**
Adds:

- Potential file structure for our main physiopy website.
- Necessary configuration files.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# NeuroImaging PREProcessing toolS (NiPreps)
```
Comment (smoia) [inline docs/index.md:1]: ?

**Review by smoia — CHANGES_REQUESTED** [review]
Ehm... I think there's a file to check :sweat_smile:

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,116 @@
+CC0 1.0 Universal
```
Comment (smoia) [inline LICENSE:1]: Can you change this license to CC BY 4.0 instead?
It's [here](https://choosealicense.com/licenses/cc-by-4.0/)

**Review by smoia — CHANGES_REQUESTED** [review]
Also, could we use a different license (basically the media version of Apache 2.0)?

File: docs/index.md (hunk 1 above)
Comment (eurunuela) [reply docs/index.md:1]: Good catch! Haha

**Review by smoia — APPROVED** [review]
AFAIK, LGTM!

**Comment by eurunuela** [comment]
Merging so @sangfrois can work on it!

**Pull request merged**